<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageDatabase</class>
 <widget class="QWidget" name="ManageDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageDatabase</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="add_employer">
     <property name="title">
      <string>Add Employer</string>
     </property>
     <widget class="QLineEdit" name="add_name_r">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_surname_r">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_employer_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Employer</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_r">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="surname_r">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="del_employee">
     <property name="title">
      <string>Delete Employee</string>
     </property>
     <widget class="QLineEdit" name="del_name_e">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="del_surname_e">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="del_employee_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Employee</string>
      </property>
     </widget>
     <widget class="QLabel" name="surname_e_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_e_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="return_mw">
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="delete_employer">
     <property name="title">
      <string>Delete Employer</string>
     </property>
     <widget class="QLineEdit" name="del_name_r">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>59</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="del_surname_r">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="del_employer_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Employer</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_r_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="surname_r_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="add_employee_box">
     <property name="title">
      <string>Add Employee</string>
     </property>
     <widget class="QLineEdit" name="add_name_e">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_surname_e">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_hl_e">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_salary_e">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="add_employee_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Employee</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_e">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="surname_e">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Surname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hl_e">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hours limit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="salary_e">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Salary:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
